src/System.Runtime/tests/System/GC.cs

```diff
@@ -3,39 +3,46 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-
 using Xunit;
 
-public class GCTests
+public static class GCTests
 {
+    private static unsafe bool s_is32Bits = sizeof(void*) == 4; // Skip IntPtr tests on 32-bit platforms
+
     [Fact]
-    public static void ValidCollectionGenerations()
+    public static void AddMemoryPressure_Invalid()
```

Comment:
minor nit: The rest of the test methods start with "Test". I actually prefer not having the "Test" prefix (seems redundant to me) and you seem to have alphabetized these method according to the names without the prefix. Anyways, minor point, feel free to just add "Test" to this one too.